models/video-model.js

```diff
@@ -0,0 +1,42 @@
+const mongoose=require("mongoose");
+const {Schema}=mongoose;
+const videoList=require("./video-data");
+
+const videoSchema=new Schema({
+  videoId:{
```

Comment:
as we get a `_id` attribute to uniquely identify each video document in the collection, I guess this would be redundant please correct me if I'm wrong

Reply:
this is a unique id which we have taken from youtube and help in playing video. This is different from the id assigned by MongoDB